lib/src/window_manager.dart

```diff
@@ -69,6 +69,19 @@ class WindowManager {
         kWindowEventLeaveFullScreen: listener.onWindowLeaveFullScreen,
       };
       funcMap[eventName]!();
+
```

Comment:
Can this live inside of the buzz_flutter_buzz package? IF I recall correctly, we have a listener class for this events, and that way, the logic is not performed in this package, but in ours...

Reply:
I have removed a code from plugin and migrated to our application.